arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -1522,6 +1523,75 @@ def rule_gfm_1_10_3(
             )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_10_4(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.10.4] The document encoding must be set in both the XML document declaration and the HTML
+    meta element for content type.
```

Comment:
This relates to existing validation `EDINET.EC1010E`. Could they be consolidated?